examples/copy-file.c

```diff
@@ -144,6 +143,10 @@ static int copy (XcpIoQueue *queue, int in, int out, off_t inSize) {
     }
   }
 
+  assert(xcp_io_queue_get_inflight_count(queue) == 0);
+  assert(xcp_io_queue_get_pending_count(queue) == 0);
+  assert(xcp_io_queue_is_empty(queue));
```

Comment:
```c
XCP_DECL_UNUSED static inline bool xcp_io_queue_is_empty (const XcpIoQueue *queue) {
  return !queue->inflightCount && !queue->pendingCount;
}
```

I think you either keep the 2 first asserts or the last but 3 is unnecessary.

Reply:
All asserts are necessary. The implementation of these methods can change at any time.
I would like to avoid an error here.

Reply:
Makes sens :+1: